.changeset/lovely-kids-sniff.md

```diff
@@ -0,0 +1,17 @@
+---
+'@example/blog': minor
```

Comment:
I think these should be patch versions since it's not a large change, right?

Reply:
Actually we don't even ever release these so no versioning needed, let me see about fixing this so that it doesn't ask you.